Mosaicism

moonso opened the pull request:
- Adds a couple of mosaicism options on variant level
- Highlight mosaic variants in variants view
- Show mosaicism on pinned variants

NOT IMPLEMENTED
- Display mosaic variants in case report
- Show is variant has been marked mosaic in other cases

vwirta commented:
How do we identify a variant as mosaic? What caller do we use for this right now?

> On 7 Nov 2018, at 18:49, Chiara Rasi <notifications@github.com> wrote:
> 
> @northwestwitch commented on this pull request.
> 
> Testing this on my local copy. A couple of comments:
> 
> I don't like much how the mosaicism of one variant is shown on the 'variants' page:
>  <https://user-images.githubusercontent.com/28093618/48138066-c7892280-e2a3-11e8-918b-03fee2260c52.png>
> I think it would be better to show a 'M' or 'mosaic' badge among the inheritance models, of a different color than the other types of inheritance, or maybe in another narrow column by itself. Or at least pull the badge on the right side of the rank column ()
> 
> Once you tag a variant as mosaic it doesn't really show on the variant page. What about adding something in the "GT call" table?
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/Clinical-Genomics/scout/pull/1051#pullrequestreview-172523921>, or mute the thread <https://github.com/notifications/unsubscribe-auth/AChiGYU2dJHWthox92LE7O07Wk8vQxefks5usvLogaJpZM4YSLDZ>.
>

northwestwitch commented:
Hi, we don't call the mosaicism, but users can assign this tag manually at the variant page, after validation I guess..

henrikstranneheim commented:
Usually, _de novo_ variants and manual inspection in IGV and then Sanger

dnil commented on scout/server/blueprints/variants/views.py:
```diff
@@ -322,19 +322,31 @@ def variant_update(institute_id, case_name, variant_id):
+        if new_dismiss:
+            flash("Added mosaic tags: {}".format(mosaic_tags), 'info')
+
+    if variant_obj.get('mosaic_tags') and not mosaic_tags:
```
..and this little shortcut won't work gracefully with multiple save buttons. I'll get back with a fix..

dnil commented:
Ok, so here goes a rough fix for the multiple buttons cross-resetting issue. Sorry to barge in - HTH!

moonso commented:
Thank you @dnil ! So I have updated the branch and rebased on master.
Regarding your comments @northwestwitch 
1. I have moved the tag to inheritance column, good suggestion!
2. I don't agree with this comment. Just like the dismiss variant options this will show in the select tags field. That is fine I think

dnil commented:
I'll assume you have tested setting and resetting different user tags after merge! Could you also write a line or two in the documentation for the users that aren't already in the loop on this? Thanks!
